backend/tests/v5/e2e/routes/users.test.js

```diff
@@ -28,7 +28,7 @@ const { updateProfile } = require('../../../../src/v5/models/users');
 
 const { templates } = require(`${src}/utils/responseCodes`);
 
-const { loginPolicy } = require(`${src}/utils/config`);
+const config = require(`${src}/utils/config`);
```

Comment:
just add cookie domain in the destructured assignment

Reply:
Im getting the trust_proxy not the cookie domain, and if I use deconstruct I cannot set it 🤔 (check line 804) this is to be able to test it cause otherwise all IPs received from the tests look the same

Reply:
or, test it with user agent change?